apps/src/componentLibrary/tabs/_Tab.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import React, {useCallback} from 'react';
+
+import FontAwesomeV6Icon, {
+  FontAwesomeV6IconProps,
+} from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
+import moduleStyles from '@cdo/apps/componentLibrary/tabs/tabs.module.scss';
+
+export interface TabModel {
+  /** Unique value of the tab */
+  value: string;
```

Comment:
would `key` be better here, since this is meant to uniquely identify the tab? I also feel like it would make more sense to see a change handler that looks like `onTabSelected: (tabKey: string) => void` vs `onChange: (value: string) => void`

Reply:
Won't it be to 'repeating'? (passing `onTabSelected` into `Tabs` component with `TabModel`) I thought we decided not to use repitative names for props/variables to make it easier to read and more unified/standardized naming.
Also, `value` is also meant to be unique and it's using here in the same way it's used in other deisgn system components. I feel like `key` might be to confusing react's `key` prop and `tabKey` might be to 'repeating' name